modules/op-setting-up-openshift-pipelines-to-download-or-view-sboms.adoc

```diff
@@ -0,0 +1,181 @@
+// This module is included in the following assemblies:
+// * secure/setting-up-openshift-pipelines-to-view-software-supply-chain-security-elements.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="op-setting-up-openshift-pipelines-to-download-or-view-sboms_{context}"]
+= Setting up Openshift Pipelines to download or view SBOMs
+
+The PipelineRun details page provides an option to download or view SBOMs, enhancing transparency and control within your supply chain. SBOMs lists all the software libraries that a component uses. Those libraries can enable specific functionality or facilitate development.
```

Comment:
I would add "in the OpenShift Web console" or something similar here

Reply:
Also the style guide calls for expanding SBOM at first use

Reply:
> in the OpenShift Web console
Updated the assembly title so that we do not have to add this for every title in the assembly.